src/UI/Components/Diagram/helpers.ts

```diff
@@ -257,17 +259,17 @@ const doesElementIsEmbeddedWithExhaustedConnections = (
  * Function that will merge state from Instance Composer to proper object for order_api endpoint.
  * Instance composer state is being split into multiple objects that could be embedded into other available, so we need to recursively
  * go through all of them to group, and sort them
- * @param {InstanceForApi[]} instances all of the instances that were created/edited in the instance, not including the one passed in second parameter
- * @param {InstanceForApi} instance instance which is used taken into consideration as the parent of the possible embedded
+ * @param {ComposerServiceOrderItem[]} instances all of the instances that were created/edited in the instance, not including the one passed in second parameter
+ * @param {ComposerServiceOrderItem} instance instance which is used taken into consideration as the parent of the possible embedded
```

Comment:
The comment is confusing me. Can you update it? Is instance then the right naming ? Seems not specific enough based on the comment